packaging/docker/build-eks-docker.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+set -euxo pipefail
+
+DOCKER_ROOT=$(realpath $(dirname ${BASH_SOURCE[0]}))
+BUILD_OUTPUT=$(realpath ${DOCKER_ROOT}/../..)
+
+echo Docker root:  $DOCKER_ROOT
+echo Build output: $BUILD_OUTPUT
+
+cd ${DOCKER_ROOT}
+
+## eg: CMAKE_PROJECT_VERSION:STATIC=7.0.0
+FDB_VERSION=$(grep CMAKE_PROJECT_VERSION\: ${BUILD_OUTPUT}/CMakeCache.txt | cut -d '=' -f 2)
+
+# Options (passed via environment variables)
+
+# Feel free to customize the image tag.
+# TODO: add a mechanism to set TAG=FDB_VERSION when we're building public releases.
+TAG=${TAG:-${FDB_VERSION}-${OKTETO_NAME}}
+ECR=${ECR:-112664522426.dkr.ecr.us-west-2.amazonaws.com}
+
+echo Building with tag ${TAG}
+
+# Login to ECR
+# TODO: Move this to a common place instead of repeatedly copy-pasting it.
+aws ecr get-login-password | docker login --username AWS --password-stdin ${ECR}
+
+docker pull ${ECR}/amazonlinux:2.0.20210326.0
+docker tag ${ECR}/amazonlinux:2.0.20210326.0 amazonlinux:2.0.20210326.0
+
+
```

Comment:
Funky whitespace is because I want this script to line up when displayed side-by-side with the (mostly copy pasted) build-release-docker.sh.